Documentation/ApiOverview/Backend/UriBuilder.rst

```diff
@@ -1,154 +1,128 @@
 :navigation-title: UriBuilder
 
-.. include:: /Includes.rst.txt
+..  include:: /Includes.rst.txt
 
-.. _edit-links:
+..  _edit-links:
 
-=====================
-Links to Edit Records
-=====================
+=====================================================
+Use the backend UriBuilder to link to "Edit Records"
+=====================================================
 
 It is often needed to create links to edit records in the TYPO3 backend.
 The same syntax is also used for creating new records.
 TYPO3 provides an API for creating such links, namely
 :code:`\TYPO3\CMS\Backend\Routing\UriBuilder`.
 
-.. hint::
+..  hint::
 
    Make sure to use :code:`\TYPO3\CMS\Backend\Routing\UriBuilder` to create
    backend links and not :code:`\TYPO3\CMS\Extbase\Mvc\Web\Routing\UriBuilder`.
 
    The variable available as :code:`$this->uriBuilder` in a controller is the
    web routing UriBuilder and can only be used for frontend links.
```

Comment:
The `hint` is indented with 4 spaces. Would be good to keep the following indented with 4 spaces, too.

```suggestion
    Make sure to use :code:`\TYPO3\CMS\Backend\Routing\UriBuilder` to create
    backend links and not :code:`\TYPO3\CMS\Extbase\Mvc\Web\Routing\UriBuilder`.

    The variable available as :code:`$this->uriBuilder` in a controller is the
    web routing UriBuilder and can only be used for frontend links.
```